lib/vitalsigns/command.rb

```diff
@@ -2,15 +2,25 @@
 require 'json'
 
 module Vitalsigns
+  # Base class for commands
   class Command
-    def execute(command, options={})
-      Open3.popen3(command) do |stdin, stdout, stderr, wait_thr|
-        return stdout.read
+    def execute(command, _options = {})
+      Open3.popen3(command) do |_stdin, stdout, _stderr, _wait_thr|
+        return parse(stdout)
       end
     end
 
     def parse(output)
-      puts output
+      output.read
     end
   end
-end
\ No newline at end of file
+
+  def self.find_device(dire_path = '/var')
```

Comment:
Yes, this can be but my thought behind adding this method under the module instead of any class is that this method is helper method for other classes.
I guess instead of creating object from class and then access method, it is better to have module method. 
If there are multiple helper methods then we can extract them into a separate module as you mentioned.

Please let me know your views on this.